src/components/SettingsPanel.vue

```diff
@@ -23,21 +23,19 @@
       </div>
       <div>
         <label
-          class="settings-panel--text"
-          @mouseover="help('displaySizes')"
-          :title="$t('settingsPanel.displaySizes.title')"
-          >{{ $t('settingsPanel.displaySizes.label') }}</label
+          class="settings-panel--legends"
+          @mouseover="help('legends')"
+          :title="$t('settingsPanel.legends.title')"
+          >{{ $t('settingsPanel.legends.title') }}</label
         >
       </div>
       <div>
-        <toggle-button
-          id="setting-toggle-display-size"
-          :value="displaySizes"
-          :width="defaultWidth"
-          :sync="true"
-          :labels="labels"
-          @change="toggleDisplaySizes"
-        />
+        <select id="setting-panel-legends" v-model="keyLegends">
+          <option value="keymap">Keymap</option>
```

Comment:
these labels need to go into i18n too. Key them off the values. Right now the UI is the where the values are coming from. I would prefer the values come out of the store rather than be hard coded.